mussidae/scripts/convert_alexs_data_to_ranges.py

```diff
@@ -0,0 +1,38 @@
+"""Conver Alex's crowd sourced data to fishing / nonfishing Ranges
+
+
+"""
+from __future__ import division
+import datetime
+import csv
+import pytz
+from mussidae import time_range_tools as trtools
+
+
+# Using a full hour for the time range would result in duplicate point issues at 
+# at the boundaries
+almost_one_hour = datetime.timedelta(minutes=59)
```

Comment:
So: what happens to points that fall in minute 59 of the hour? Can this be minutes=59,seconds=59, millis=9999?